### DISCUSSION TITLE
Persistent Window Size and Position

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9575] [CLI - Electron mode]
Does anyone know of a way in which I can store the window size and it's location so when the users close and reopen the application it's position and size are remembered?

This is a nice feature i see in a lot of desktop apps including ones I've written in C++ and Qt. However I don't quite know how to do this in electron with quasar after doing some initial research i came up empty.

Thanks

**Comment by ontwikkelfabriek** [comment]
@JokerMartini 

Here's some info:
https://stackoverflow.com/questions/51328586/how-to-restore-default-window-size-in-an-electron-app
